datastore/adapter/tiered.py

```diff
@@ -49,7 +49,7 @@ class _Adapter(_support.DatastoreCollectionMixin[DS], typing.Generic[DS]):
 		stores: typing.List[DS] = self._stores.copy()
 		for store in stores:
 			try:
-				value_: RT = await store.get(key)  # type: ignore[assignment]
+				value_: RT = await store.get(key)  # type: ignore
```

Comment:
Why drop the category label here? (I try to be as specific as possible with mypy typing exceptions, so that potential other problems in the same line aren't missed.)

Reply:
flake8 was giving 
```
datastore/core/util/stream.py:844:26: F821 undefined name 'ignore'
datastore/core/util/stream.py:844:26: F821 undefined name 'union'
datastore/core/util/stream.py:844:26: F821 undefined name 'attr'
```

I can do a `# noqa: F821` on all these instances if you think that's preferred.